tests/testrelayv2.nim

```diff
@@ -301,9 +298,7 @@ suite "Circuit Relay V2":
     philosophical flourish Cato throws himself upon his sword; I quietly
     take to the ship."""
             )
-          except CancelledError as e:
-            raise e
-          except CatchableError:
+          except LPStreamError:
             discard # will get here after data exceeded
```

Comment:
nit: would be better to rewrite test, last `writeLp`  should be used in `expect LPStreamError` block, because that one should get error